src/extension/app-schema/app-schema-test/src/test/resources/test-data/stations/layerNamesTest/measurementsDefaultGeometry.properties

```diff
@@ -0,0 +1,4 @@
+_=ID:String,NAME:String,UNIT:String,STATION:String,SAMPLEDAREA_CODE:String,SAMPLEDAREA_GEOM:Geometry:srid=4326
```

Comment:
What do ``DefaultGeometry`` tests cases have to do with this ? The stations use case base properties should be enough for this no ?